kolibri/plugins/learn/assets/src/views/content-page/index.vue

```diff
@@ -135,7 +137,7 @@
       licenseDescription: 'License description',
       copyrightHolder: 'Copyright holder: {copyrightHolder}',
     },
-    data: () => ({ wasIncomplete: false }),
+    data: () => ({ wasIncomplete: false, showLicenceDescription: false }),
```

Comment:
minor, but `showLicenceDescription` sounds like a method name. Sometimes I like to start booleans with `is`, like `isLicenseVisible`?